Assets/MixedRealityToolkit/Services/MixedRealityToolkit.cs

```diff
@@ -848,7 +820,7 @@ private bool RegisterServiceInternal(Type interfaceType, IMixedRealityService se
             if (!isInitializing)
             {
                 serviceInstance.Initialize();
-                serviceInstance.Enable();
+                //serviceInstance.Enable();
```

Comment:
Is this supposed to be re-enabled or removed?

Reply:
This addresses #4832. I'll remove the comment.